app/buyers/views/buyers.py

```diff
@@ -362,6 +363,11 @@ def publish_brief(framework_slug, lot_slug, brief_id):
             question_and_answers['slug'] = section['id']
 
     unanswered_required, unanswered_optional = count_unanswered_questions(sections)
+
+    if sections.get_section('set-how-long-your-requirements-will-be-live-for') and \
+            sections.get_section('set-how-long-your-requirements-will-be-live-for').questions[0].answer_required:
+            unanswered_required -= 1
```

Comment:
Cool, will add a comment. There's a bit of an explanation is the comment for this 1993e437d74d50f2f18dc6718db864eb2afc7398  commit which will help too